docs/references/vod/upload.mdx

```diff
@@ -60,12 +110,38 @@ curl --location --request POST 'https://livepeer.studio/api/asset/request-upload
 Now upload (`PUT`) a video file on the exact URL in the `url` field of the
 response above:
 
-```jsx
+
+
+<Tabs>
+  <TabItem value="bash" label="bash">
+
+```bash
 curl --location --request PUT "${url}" \
 --header 'Content-Type: video/mp4' \
 --data-binary '@$VIDEO_FILE_PATH'
 ```
 
+</TabItem>
+
+<TabItem value="py" label="Fetch">
+
+```js
+ fetch(`${assetURL}`, {
+        method: "PUT",
+        headers: {
+          "Content-Type": "video/mp4",
+        },
+        body: file, // Asset for upload
+      });
+  .then((res) => {
+    console.log(JSON.stringify(res.data));
+  })
+  .catch((error) => console.log(error));
```

Comment:
Don't need to change this now, but please try to use `async/await` whenever possible. Code with `then/catch` feels unnecessarily polluted and is harder to write and to understand what's going on. That's the old way of writing JavaScript, back when there was only the callback hell everywhere. Async/await FTW! 😆

Reply:
@victorges  please review the changes of the latest commits:
https://github.com/livepeer/livepeer-studio-docs/pull/144/commits/7723461a2cebe47d3b76ac8df9f44ea760c18e38
https://github.com/livepeer/livepeer-studio-docs/pull/144/commits/f15deb6dd85aa973ea0cc01b8fdefd1b8042778f

Reply:
I just left them out since but could put them back in if you want?

Reply:
No need to! We should probably shift towards cleaner code examples. Users can add some logs when they wanna start debugging.